content/docs/command-reference/init.md

```diff
@@ -7,19 +7,29 @@ Initialize a MLEM project.
 ```usage
 usage: mlem init [options] [path]
 
-arguments: [PATH] Where to init project
+arguments:
+[path]      Location (file path or URL) to initialize a MLEM project
 ```
 
 ## Description
 
-The `init` command (without given `path`) defaults to the current directory for
-the path argument. This creates a `.mlem/` directory and an empty `config.yaml`
-file inside it.
+This creates a `.mlem/` directory and an empty `config.yaml` file in the desired
+project `path`, which defaults to the current working directory (`.`).
 
-Although we recommend using MLEM within a Git repository to track changes using
-the standard Git workflows, this is not required. The existence of a `.mlem/`
-directory in any path (including remote) constitutes a MLEM project, and MLEM
-will be fully functional even without incorporating Git in one's workflow.
+The existence of a valid `.mlem/` directory in any location (including [remote])
+allows MLEM to store a references to MLEM objects stored in the project,
+enabling `$ mlem list` command, and configure MLEM
+[to work with DVC](http://localhost:8000/doc/use-cases/dvc) by setting this in
+`.mlem/config.yaml`.
+
+<admon type="tip">
+
+We recommend initializing MLEM projects inside Git repositories to track changes
+using standard Git workflows.
```

Comment:
What was the idea behind this? What Git workflows you assumed this enables? @jorgeorpinel

Reply:
> What was the idea behind this?

@aguschin I just moved this sentence from the previous paragraph into an admonition. Currently it reads

> Although we recommend using MLEM within a Git repository to track changes using
the standard Git workflows, ...

Reply:
Ok, I think I got it now. I think it's clearer to say it like that. WDYT?
```suggestion
We recommend initializing MLEM projects inside Git repositories to track changes and
act on them using standard Git workflows.
```